tests/CatalogTests/Registration/RegistrationMakerCatalogItemTests.cs

```diff
@@ -81,5 +82,99 @@ private static string GetContentString(StorageContent content)
+                                                                             string expectedLicenseUrlValue, string expectedLicenseExpressionValue, string galleryBaseAddress)
+            {
+                // Arrange
+                if (hasLicenseUrl)
+                {
+                    _graph.Assert(
+                    _graph.CreateUriNode(_catalogUri),
+                    _graph.CreateUriNode(Schema.Predicates.LicenseUrl),
+                    _graph.CreateLiteralNode("https://test.org"));
+                }
+                if (hasLicenseExpression)
+                {
+                    _graph.Assert(
+                    _graph.CreateUriNode(_catalogUri),
+                    _graph.CreateUriNode(Schema.Predicates.LicenseExpression),
+                    _graph.CreateLiteralNode("MIT"));
+                }
+                if (hasLicenseFile)
+                {
+                    _graph.Assert(
+                    _graph.CreateUriNode(_catalogUri),
+                    _graph.CreateUriNode(Schema.Predicates.LicenseFile),
+                    _graph.CreateLiteralNode("license"));
+                }
+
+                var item = new RegistrationMakerCatalogItem(
+                    _catalogUri,
+                    _graph,
+                    _registrationBaseAddress,
+                    isExistingItem: false,
+                    packageContentBaseAddress: _packageContentBaseAddress,
+                    galleryBaseAddress: new Uri(galleryBaseAddress))
+                {
+                    BaseAddress = _baseAddress,
+                };
+                RegistrationMakerCatalogItem.PackagePathProvider = new PackagesFolderPackagePathProvider();
+                var context = new CatalogContext();
+
+                // Act
+                var content = item.CreatePageContent(context);
+                var licenseUrlTriples = content.GetTriplesWithSubjectPredicate(
+                    content.CreateUriNode(_catalogUri),
+                    content.CreateUriNode(Schema.Predicates.LicenseUrl));
+                var licenseExpressionTriples = content.GetTriplesWithSubjectPredicate(
+                    content.CreateUriNode(_catalogUri),
+                    content.CreateUriNode(Schema.Predicates.LicenseExpression));
+                var licenseFileTriples = content.GetTriplesWithSubjectPredicate(
+                    content.CreateUriNode(_catalogUri),
+                    content.CreateUriNode(Schema.Predicates.LicenseFile));
+
+                // Assert
+                Assert.Equal(1, licenseUrlTriples.Count());
+                Assert.Equal(expectedLicenseUrlValue, licenseUrlTriples.First().Object.ToString());
+                Assert.Equal(1, licenseExpressionTriples.Count());
+                Assert.Equal(expectedLicenseExpressionValue, licenseExpressionTriples.First().Object.ToString());
```

Comment:
Is there a way not to generate `licenseExpression` in the output if it is not present in the input instead of generating an empty one?

Reply:
Keep the empty string now.